bom/runtime/pom.xml

```diff
@@ -538,6 +540,11 @@
                 <artifactId>quarkus-keycloak-admin-client-deployment</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-kafka-client</artifactId>
+                <version>${project.version}</version>
```

Comment:
This looks totally unrelated. Is it missing from the BOM? If so, should it be added in another PR that we can backport?